docs/content/development/build.md

```diff
@@ -2,28 +2,70 @@
 layout: doc_page
 ---
 
-### Build from Source
-
-You can build Druid directly from source. Please note that these instructions are for building the latest stable of Druid. 
+You can build Druid directly from source. Please note that these instructions are for building the latest stable version of Druid.
 For building the latest code in master, follow the instructions [here](https://github.com/druid-io/druid/blob/master/docs/content/development/build.md).
 
+# Requirements
+
 Building Druid requires the following:
-- [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-- [Maven version 3.x](http://maven.apache.org/download.cgi)
 
-To do so, run these commands:
+- [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)+
+- [Maven 3.x](http://maven.apache.org/download.cgi)+
+
+# Building
+
+## Build a runtime distribution
+
+Move to the project root directory and run:
+
+```
+mvn clean package -DskipTests
+```
+
+This will compile the project and create the Druid binary distribution tarball under `distribution/target/druid-VERSION-bin.tar.gz`
+and `mysql-metadata-storage` extension under `distribution/target/mysql-metadata-storage-bin.tar.gz`.
+
+- For more information on `mysql-metadata-storage` extension, see [here](../development/extensions-core/mysql.html).
+- For more information on including extensions, see [here](../operations/including-extensions.html).
+
+## Build javadoc
+
+```
+mvn install -DskipTests && mvn javadoc:javadoc
+```
+
+# Testing
+
+## Run all unit/integration tests
```

Comment:
The below command doesn't run integration tests. We already have a document for integration tests (https://github.com/apache/incubator-druid/blob/master/integration-tests/README.md), so it would be better add a link to it.